string relay
bool state
---
bool boardExists
bool foundRelay
bool toggleSuccess
